### ISSUE TITLE
Alert: Color accessibility issue

### ISSUE THREAD

**Issue by gabriel-piltzer-trimble** [trimble-oss/modus-wc-2.0#637]
### Prerequisites

- [x] I have [searched](https://github.com/trimble-oss/modus-wc-2.0/issues?q=is%3Aissue) for duplicate or closed issues
- [x] I have read the [contributing guidelines](https://github.com/trimble-oss/modus-wc-2.0/blob/main/CONTRIBUTING.md)

### Describe the issue

The Alert component suffers from accessibility issues and does not meet minimum color contrast thresholds. I tested this for the success variant but I believe it also applies to error and warning as well.

<img width="2545" height="1270" alt="Image" src="https://github.com/user-attachments/assets/9f57b22b-42ed-4a69-b265-0d59fed5cb35" />

### Reduced test cases

1. Install axe DevTools Chrome browser extension
2. Open [Alert](https://trimble-oss.github.io/modus-wc-2.0/prod/?path=/docs/components-alert--docs&globals=theme:modus-modern-dark) page with theme modus-modern-dark
3. Select the success variant (probably exists for error and warning also)
3. Open axe DevTools and run a full page scan

### What is the issue regarding?

@trimble-oss/modus-wc

### Priority

Medium

### What product/project are you using Modus Web Components for?

Trimble Finance Assistant and Prism

### What is your team/division name?

CMS

### Are you willing to contribute?

Yes

**Comment by ElishaSamPeterPrabhu** [comment]
<img width="1692" height="170" alt="Image" src="https://github.com/user-attachments/assets/8b0ba682-de80-421d-aaa6-84a13808e0d0" />

Closing this issue, as the colors were updated in this PR ([https://github.com/trimble-oss/modus-wc-2.0/pull/635](https://github.com/trimble-oss/modus-wc-2.0/pull/635)) and are compliant with color contrast requirements.
